dle/cfn/deploy_rds.yaml

```diff
@@ -0,0 +1,40 @@
+Parameters:
+  DBTypeParameter:
+    Type: String
+    Default: "db.t4g.medium"
+    AllowedValues:
+      - db.t4g.medium
+    Description: Default is db.t4g.medium.
+  VPCParameter:
+    Type: String
+    Default: "vpc-23074059"
+    Description: VPC ID
+
+Resources:
+  globalRxDBSecGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: Open 5432
+      VpcId: !Ref VPCParameter
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: '5432'
+          ToPort: '5432'
+          CidrIp: 0.0.0.0/0 
```

Comment:
Should definitely restrict to other security groups for ECS, and IPs for us if people need to connect directly for some reason